source/mesh_refinement/composition_gradient.cc

```diff
@@ -66,16 +67,17 @@ namespace aspect
                 // composition gradient. Note that quadrature points and dofs
                 // are enumerated in the same order.
                 for (unsigned int j=0; j<this->get_fe().base_element(this->introspection().base_elements.compositional_fields).dofs_per_cell; ++j)
-                  indicators[i] += composition_gradients[j].norm() * composition_scaling_factors[c];
+                  this_indicator[i] += composition_gradients[j].norm();
 
                 // Scale gradient in each cell with the correct power of h. Otherwise,
                 // error indicators do not reduce when refined if there is a density
                 // jump. We need at least order 1 for the error not to grow when
                 // refining, so anything >1 should work. (note that the gradient
                 // itself scales like 1/h, so multiplying it with any factor h^s, s>1
                 // will yield convergence of the error indicators to zero as h->0)
-                indicators[i] *= std::pow(cell->diameter(), power);
```

Comment:
ahh..., only this one is different than others. :D  It was mentioned as in issue #1659